packages/domain-picker/test/suggestion-item.tsx

```diff
@@ -15,293 +16,76 @@ import { render, fireEvent, screen } from '@testing-library/react';
-				};
-
-				const { rerender } = render( <SuggestionItem { ...props } /> );
-
-				expect( recordAnalytics ).toHaveBeenCalledWith(
-					expect.objectContaining( {
-						railcarId: props.railcarId,
-						result: expect.stringContaining( props.suggestion.domain_name ),
-						trainTracksType: 'render',
-						uiPosition: props.uiPosition,
-					} )
-				);
-
-				recordAnalytics.mockClear();
-				props[ prop ] = value2;
-
-				rerender( <SuggestionItem { ...props } /> );
-
-				expect( recordAnalytics ).not.toHaveBeenCalled();
-			} );
+		it( 'should send a render event when domain name and railcarId changes', async () => {
+			const { rerender } = renderComponent();
+			const updatedProps = { ...MOCK_PROPS, domain: 'example1.com', railcarId: 'id1' };
+			rerender( <SuggestionItem { ...updatedProps } /> );
+
+			expect( updatedProps.onRender ).toHaveBeenCalledTimes( 2 );
 		} );
 	} );
 
 	describe( 'interact event', () => {
-		it( 'sends interact event when selected', async () => {
-			// Delay import so we have time to load configData in `beforeAll`
-			const { default: SuggestionItem } = await import( '../suggestion-item' );
-
-			const recordAnalytics = jest.fn();
-			const railcarId = 'id';
-
-			const { getByLabelText } = render(
-				<SuggestionItem
-					isSelected={ false }
-					suggestion={ testSuggestion }
-					onSelect={ jest.fn() }
-					railcarId={ railcarId }
-					recordAnalytics={ recordAnalytics }
-					uiPosition={ 113 }
-				/>
-			);
-
-			fireEvent.click(
-				// Use function matcher to deal with the label containing <spans>
-				getByLabelText( ( content, element ) => element.textContent === 'example.com' )
-			);
-
-			expect( recordAnalytics ).toHaveBeenCalledWith( {
-				trainTracksType: 'interact',
-				action: 'domain_selected',
-				railcarId,
-			} );
-		} );
+		it( 'should send an interact event when selected', async () => {
+			renderComponent();
+
+			fireEvent.click( screen.getByRole( 'button' ) );
 
-		it( "doesn't send interact event when deselected", async () => {
-			// Delay import so we have time to load configData in `beforeAll`
-			const { default: SuggestionItem } = await import( '../suggestion-item' );
-
-			const recordAnalytics = jest.fn();
-
-			const { getByLabelText } = render(
-				<SuggestionItem
-					isSelected={ true }
-					suggestion={ testSuggestion }
-					onSelect={ jest.fn() }
-					railcarId="id"
-					recordAnalytics={ recordAnalytics }
-					uiPosition={ 113 }
-				/>
-			);
-
-			fireEvent.click(
-				// Use function matcher to deal with the label containing <spans>
-				getByLabelText( ( content, element ) => element.textContent === 'example.com' )
-			);
-
-			expect( recordAnalytics ).not.toHaveBeenCalledWith(
-				expect.objectContaining( {
-					trainTracksType: 'interact',
-				} )
-			);
+			expect( MOCK_PROPS.onSelect ).toHaveBeenCalledWith( MOCK_PROPS.domain );
 		} );
 	} );
 } );
-/* eslint-enable */
 
 describe( 'check conditional elements render correctly', () => {
```

Comment:
in line with the rest of the file, this describe call could be renamed `conditional elements` ?